core/server/services/auth/session/store.js

```diff
@@ -0,0 +1,44 @@
+const {Store} = require('express-session');
+const common = require('../../../lib/common');
+
+module.exports = class SessionStore extends Store {
+    constructor(SessionModel) {
+        super();
+        this.SessionModel = SessionModel;
+    }
+
+    destroy(sid, callback) {
```

Comment:
Is it required (from the session store) to use callbacks here? 🤔

Reply:
Yup that's the interface required

Reply:
Ok 👍 Thx